src/main/resources/com/google/api/codegen/go/doc.snip

```diff
@@ -10,6 +10,16 @@
     @join line : view.packageDoc
         // {@line}
     @end
+    //
+    // Use of Context
+    //
+    // NewClient and all client methods require a context. NewClient context is
```

Comment:
The ctx passed to NewClient is used for authentication requests and gRPC transport setup, but not for future calls performed using the client. These calls use the ctx passed to individual methods on the client.